src/widgets/required/ui/required.tsx

```diff
@@ -43,7 +43,7 @@ export const Required = ({ courseName, marked1, marked2 }: RequiredProps) => {
               ? willLearn.map((willLearn) => {
                 return (
                   <article key={willLearn.title}>
-                    <Subtitle text={willLearn.title} />
+                    <Subtitle>{willLearn.title}</Subtitle>
```

Comment:
This doesn't seem to be working. Let's remove it to make it consistent too.

![image](https://github.com/user-attachments/assets/374be403-5c37-44b7-8b44-27731fbe8399)